crates/miden-lib/src/transaction/memory.rs

```diff
@@ -22,7 +22,7 @@ pub type StorageSlot = u8;
 // | Partial blockchain | 1_200 (300)                           | 1_331? (332?)                       |                                             |
 // | Kernel data        | 1_600 (400)                           | 1_739 (434)                         | 34 procedures in total, 4 elements each     |
 // | Accounts data      | 8_192 (2048)                          | 532_479 (133_119)                   | 64 accounts max, 8192 elements each         |
-// | Account delta      | 532_480 (133_120)                     | TODO (TODO)                         |                                             |
+// | Account delta      | 532_480 (133_120)                     | 533_511 (133_377)                   |                                             |
```

Comment:
Is this correct? According to the table on lines 55 - 60 below, we need 267 elements to represent account delta. But `532_480 + 267 = 532_747` and not `533_511`. Or am I missing something?

Reply:
Thanks, that was still using the WORD_SIZE storage map pointers.